test/paper-input.html

```diff
@@ -162,6 +162,37 @@
 
     });
 
+    suite('focus/blur events', function() {
+      var input;
+
+      setup(function() {
+        input = fixture('basic');
+      });
+
+      test('focus/blur events fired on host element', function(done) {
```

Comment:
If `IronControlState` implements and tests this behavior, is it useful for this to also test the behavior?